src/header.md

```diff
@@ -2,5 +2,83 @@ Welcome to YouPin API documents.
 
 This document explains how you can create/retrive/modify/delete information in YouPin database.
 
-To get started, you need to ask YouPin team to create 3rd-party app account for you.
-The team will provide you an app id and key that you will use to access YouPin API.
+To retrieve information from YouPin API, you are ready to go.
+But if you want to create/modify/delete data, you need to ask YouPin team to create user account for you.
```

Comment:
How to contact YouPin team? Email? 

Also note that  we will let developers create accounts by themselves in the future.